csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.shared.cs

```diff
@@ -821,7 +821,9 @@ static NativeMethods()
         // CoreML is available on iOS and macOS so we can't exclude based on __MOBILE__ && __IOS__
```

Comment:
just curious - does `!__ANDROID__` mean iOS or MacOS?

Reply:
We don't have a way to separate out MacOS. The Android and iOS values are determined by the target framework as there are separate Xamarin frameworks for each. MacOS doesn't have a separate .net framework.